WalletWasabi.Tests/P2pTests.cs

```diff
@@ -79,31 +79,29 @@ public async Task TestServicesAsync(string networkString)
 				addressManager = new AddressManager();
 			}
 
+			BitcoinStore bitcoinStore = new BitcoinStore();
+			await bitcoinStore.InitializeAsync(Path.Combine(Global.Instance.DataDir, nameof(TestServicesAsync)), network);
```

Comment:
Shouldn't we not use nameof here?

Reply:
No, why? The test's workfolder should keep up with the changes of test's name.